doc/source/contributor-tutorial-contribute-on-github.rst

```diff
@@ -256,36 +255,36 @@ Example of first contribution
 Problem
 *******
 
-For our documentation, we’ve started to use the `Diàtaxis framework <https://diataxis.fr/>`_.
+For our documentation, we've started to use the `Diàtaxis framework <https://diataxis.fr/>`_.
 
-Our “How to” guides should have titles that continue the sencence “How to …”, for example, “How to upgrade to Flower 1.0”.
+Our "How to" guides should have titles that continue the sentence "How to …", for example, "How to upgrade to Flower 1.0".
 
 Most of our guides do not follow this new format yet, and changing their title is (unfortunately) more involved than one might think.
 
-This issue is about changing the title of a doc from present continious to present simple.
+This issue is about changing the title of a doc from present continuous to present simple.
 
-Let's take the example of “Saving Progress” which we changed to “Save Progress”. Does this pass our check?
+Let's take the example of "Saving Progress" which we changed to "Save Progress". Does this pass our check?
 
-Before: ”How to saving progress” ❌
+Before: "How to saving progress" ❌
 
-After: ”How to save progress” ✅
+After: "How to save progress" ✅
 
 Solution
 ********
 
-This is a tiny change, but it’ll allow us to test your end-to-end setup. After cloning and setting up the Flower repo, here’s what you should do:
+This is a tiny change, but it'll allow us to test your end-to-end setup. After cloning and setting up the Flower repo, here's what you should do:
 
 - Find the source file in ``doc/source``
 - Make the change in the ``.rst`` file (beware, the dashes under the title should be the same length as the title itself)
-- Build the docs and check the result: `<https://flower.ai/docs/writing-documentation.html#edit-an-existing-page>`_
+- Build the docs and `check the result <contributor-how-to-write-documentation.html#edit-an-existing-page>`_
```

Comment:
Should this also have `:doc:`?
```suggestion
- Build the docs and :doc:`check the result <contributor-how-to-write-documentation.html#edit-an-existing-page>`_
```

Reply:
The problem with `:doc:` is that it can only refer to a document as a whole (so to the whole contributor-how-to-write-documentation page in this case), not to specific sections within that page (the edit-an-existing-page section in this case). 
That would require `:ref:` and a specific anchor on that section in the page. Personally I think that would be a good idea (even better than using `:doc:` in general), since even with name changes of documents the reference will always stay correct, or give an error if the section is removed. 

I didn't implement `:ref:` as I would need to define anchor names, but didn't see any guidelines for that. But if you think it would be better to go for `:ref:` everywhere from the start I can implement it.

Reply:
Thanks for the explanation. I'm resolving the comment, I think we can merge this as-is.